frontend/public/locales/en/translation.json

```diff
@@ -363,12 +363,13 @@
     "clusterPool.creation.success.message": "{{name}} was successfully created.",
     "clusterPool.creation.success.title": "Cluster pool created",
     "clusterPool.destroy": "Destroy cluster pool",
-    "clusterPool.modal.scale.input": "Set desired cluster pool size",
-    "clusterPool.modal.scale.message": "Specify the desired size of <bold>{{clusterPoolName}}</bold>. Adjusting the size of the cluster pool will result in the creation or destruction of clusters; only unclaimed clusters will be destroyed if downsizing.",
+    "clusterPool.modal.scale.message": "Specify the desired size of <bold>{{clusterPoolName}}</bold>. Adjusting the size or running count of the cluster pool will result in the creation or destruction of clusters; only unclaimed clusters will be destroyed if downsizing.",
```

Comment:
I'm not sure if it is accurate to say that adjusting the running count will result in the creation or destruction of clusters. Usually it should just affect how many are running vs. hibernating. I think we confirmed that if `running` is greater than `size`, you just get `size` running clusters. And when reducing `runningCount` I would think that running cluster are then just hibernated. Please check and then clarify.

Reply:
This is correct, just fixed the message